homework_7.py

```diff
@@ -0,0 +1,120 @@
+""""HOMEWORK #7. Andrei Kulbitski"""
+
+import random
+
+# 1. Быки и коровы¶
+print("TASK 1: BULLS AND COWS")
+
+
+def magic_number_generator_func():
+    """Random unique 4-digit generator"""
+    digits = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0]
+    random.shuffle(digits)
+    return digits[:4]
+
+
+# Magic number is stored as four integers packaged in the list
+magic_number = magic_number_generator_func()
+
+# User is expected to guess the number in four attempts :)
+attempts = 8
```

Comment:
can not this information in the requirements

Reply:
attempts removed